runner/hooks/completed.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# shellcheck source=runner/logger.bash
+source logger.bash
+
+log.notice "Running ARC Completed Hooks"
+
+for hook in $(ls /etc/arc/hooks/completed.d); do
```

Comment:
```suggestion
for hook in $(ls /etc/arc/hooks/job_completed.d); do
```

I prefer having the `job_` prefix. Same for `completed.sh`